src/core/Makefile.am

```diff
@@ -89,6 +87,13 @@ SOURCES_COMMON                      = \
     utils/slaac_address.cpp           \
     $(NULL)
 
+if OPENTHREAD_ENABLE_MAC_WHITELIST
```

Comment:
Can you add all these files to the Windows build and enable these features by default?